test/verify/check-kdump

```
@@ -559,13 +559,13 @@ class TestKdumpNFS(KdumpHelpers):
 
 
 @testlib.skipOstree("kexec-tools not installed")
-@testlib.onlyImage("Only Fedora has ansible on the image", "fedora-*")
+@testlib.onlyImage("Only Fedora/RHEL has ansible on the image", "fedora-*", "rhel-9-7")
```

Please "rhel-9-*" os that it doesn't silently skip again once we move to 9-8.